app/_src/gateway/kong-manager/auth/ldap/configure.md

```diff
@@ -50,8 +50,12 @@ Attribute | Description
 The **Sessions plugin** (configured with `admin_gui_session_conf`) requires a secret and is configured securely by default.
 * Under all circumstances, the `secret` must be manually set to a string.
 * If using HTTP instead of HTTPS, `cookie_secure` must be manually set to `false`.
+{% if_version lte:3.1.x %}
 * If using different domains for the Admin API and Kong Manager, `cookie_samesite` must be set to `off`.
-Learn more about these properties in [Session Security in Kong Manager](/gateway/{{page.kong_version}}/kong-manager/auth/sessions/#session-security), and see [example configurations](/gateway/{{page.kong_version}}/kong-manager/auth/sessions/#example-configurations).
+{% endif_version %}
+{% if_version gte:3.2.x %}
+* If using different domains for the Admin API and Kong Manager, `cookie_same_site` must be set to `off`.
+{% endif_version %}Learn more about these properties in [Session Security in Kong Manager](/gateway/{{page.kong_version}}/kong-manager/auth/sessions/#session-security), and see [example configurations](/gateway/{{page.kong_version}}/kong-manager/auth/sessions/#example-configurations).
```

Comment:
```suggestion
{% endif_version %}

Learn more about these properties in [Session Security in Kong Manager](/gateway/{{page.kong_version}}/kong-manager/auth/sessions/#session-security), and see [example configurations](/gateway/{{page.kong_version}}/kong-manager/auth/sessions/#example-configurations).
```
I think? I think this applies to all the bullets.

Reply:
I originally did that because that point is formatted like this in older content too, eg: https://deploy-preview-5183--kongdocs.netlify.app/gateway/2.8.x/configure/auth/kong-manager/ldap/

But - you're right. The info applies to all bullets in the list, not to this particular one. It was formatted incorrectly in the original doc - nice catch.